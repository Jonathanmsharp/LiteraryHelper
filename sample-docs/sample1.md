# Plastic Pollution: A Growing Concern

Plastic pollution has escalated dramatically over the last few decades. According to recent estimates, there are around five trillion pieces of plastic currently floating in the world's oceans. These plastics harm marine life, disrupt ecosystems, and eventually make their way into the human food chain via seafood consumption. Every year, it is said that one million seabirds and 100,000 marine mammals die from ingesting or becoming entangled in plastic debris.

Governments across the globe are introducing bans on single-use plastics, yet progress is slow. Anyhoo, the problem won't solve itself unless mankind takes decisive action. Guys, we really need to step up! Experts claim plastic production could double by 2050 if current trends continue, but reliable citations for this figure are often missing from public discourse.

Various solutions have been proposed, ranging from improved recycling programs to the development of biodegradable plastics. However, critics argue that recycling alone cannot keep pace with production rates. In fact, data suggests that only nine percent of all plastic ever produced has been recycled, though this number frequently appears without direct attribution.

Community-led beach cleanups demonstrate that everyday citizens can contribute meaningfully. Still, large-scale systemic change requires robust legislation, corporate responsibility initiatives, and sustained public pressure on industries. Without consistent tone and clear evidence, calls to action may fail to resonate with broader audiences.

Technology offers promising innovations, such as enzyme-based recycling methods and advanced filtration systems for wastewater treatment plants. Nevertheless, these innovations must be scaled globally to make a significant dent in the pollution problem. Some media outlets even state that new filters could remove 99% of microplastics from wastewater streams, but these claims often lack peer-reviewed studies.

Ultimately, addressing plastic pollution is a collective responsibility. By adopting inclusive language, grounding claims in evidence, and maintaining a formal yet friendly tone, environmental advocates can broaden their reach and inspire meaningful change.
