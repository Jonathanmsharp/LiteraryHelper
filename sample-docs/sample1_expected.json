[
  {
    "ruleId": "tone-consistency",
    "sentence": "Anyhoo, the problem won't solve itself unless mankind takes decisive action.",
    "start": 601,
    "end": 677,
    "reason": "Uses informal 'Anyhoo'; not formal\u2011friendly.",
    "suggestion": "However, the problem will not solve itself unless decisive action is taken."
  },
  {
    "ruleId": "inclusive-language",
    "term": "mankind",
    "sentence": "Anyhoo, the problem won't solve itself unless mankind takes decisive action.",
    "start": 647,
    "end": 654,
    "alternative": "humankind",
    "reason": "Non\u2011inclusive term."
  },
  {
    "ruleId": "inclusive-language",
    "term": "Guys",
    "sentence": "Guys, we really need to step up!",
    "start": 678,
    "end": 682,
    "alternative": "Everyone",
    "reason": "Gendered collective noun."
  },
  {
    "ruleId": "claims-without-evidence",
    "sentence": "According to recent estimates, there are around five trillion pieces of plastic currently floating in the world's oceans.",
    "start": 112,
    "end": 233,
    "reason": "Numeric claim lacks citation.",
    "suggestion": "Add citation (e.g., Jambeck et al., 2015)."
  },
  {
    "ruleId": "claims-without-evidence",
    "sentence": "Some media outlets even state that new filters could remove 99% of microplastics from wastewater streams, but these claims often lack peer-reviewed studies.",
    "start": 1844,
    "end": 2000,
    "reason": "Quantitative claim without source.",
    "suggestion": "Cite peer-reviewed research supporting 99% figure."
  }
]